packages/components/src/IconsProvider/IconsProvider.component.js

```diff
@@ -31,15 +64,16 @@ function IconsProvider({ defaultIcons, icons }) {
 }
 
 IconsProvider.displayName = 'IconsProvider';
-
+IconsProvider.context = context;
 IconsProvider.propTypes = {
 	defaultIcons: PropTypes.object, // eslint-disable-line react/forbid-prop-types
 	icons: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+	getIconHref: PropTypes.func, // eslint-disable-line react/forbid-prop-types
```

Comment:
You take a function but it is used as boolean only.

Reply:
I don t get the point here. It is an optional function. If I put boolean it will be always wrong